tests/toDo.test.js

```diff
@@ -0,0 +1,72 @@
+import ToDo from "../src/models/toDo";
+import ToDoList from "../src/models/toDoList";
+import { describe, it, expect } from "vitest";
+
+describe("ToDo", () => {
+  it("should create a new ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
+
+    expect(toDo.id).toBeGreaterThan(0);
+    expect(toDo.description).toBe("Buy milk");
+    expect(toDo.done).toBe(false);
```

Comment:
```suggestion
    expect(toDo.description).toBe(FORM_DEFAULT_DESCRIPTION);
    expect(toDo.done).toBe(FORM_DEFAULT_COMPLETION_STATUS);
```

constants should be declared with corresponding values.